51.md

```diff
@@ -49,6 +49,7 @@ Aside from their main identifier, the `"d"` tag, sets can optionally have a `"ti
 | Curation sets | 30004 | groups of articles picked by users as interesting and/or belonging to the same category      | `"a"` (kind:30023 articles), `"e"` (kind:1 notes)                                 |
 | Curation sets | 30005 | groups of videos picked by users as interesting and/or belonging to the same category        | `"a"` (kind:34235 videos)                                                         |
 | Interest sets | 30015 | interest topics represented by a bunch of "hashtags"                                         | `"t"` (hashtags)                                                                  |
+| Audio Playlist| 30029 | a list of kind `31337` audio tracks                                                          | `"a"` (kind:31337 audio tracks)                                                   |
```

Comment:
Maybe here add also a list kind for Podcasts (episodes with kind:31338), or maybe call this `Audio Collection` and it could be applicable to both and maybe more kinds in the future? Because an Album/Release could also be applicable... also support `title` tag and `imeta` for enclosure/cover art, and maybe description in the `.content`...